package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "telemetry-events",
-  "version": "1.0.1",
+  "version": "1.1.0",
```

Comment:
The public API has not changed at all, correct? If so, the new version should be `1.0.2`. This could also be classified as a bugfix with respect to ES6, right? Or ...is this library now being declared es6-compatible?